assets/css/styles.css

```diff
@@ -0,0 +1,568 @@
+  text-transform: uppercase;
+  color: var(--secondary-color);
+}
+
+.hero-end :last-child {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 23px;
+  color: var(--secondary-color);
+  margin-top: -1rem;
+}
+
+#main-program {
+  background-image: url(../images/pictures/gray-texture.png);
+  margin-top: -1.2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1em 1em;
+}
+
+h2 {
+  font-weight: 800;
+  font-size: 24px;
+  line-height: 29px;
+  color: var(--light-color);
+  display: flex;
+  justify-content: center;
+  text-transform: capitalize;
+}
+
+hr {
+  width: 70px;
+  border: 0.8px solid #ec5242;
+  margin: 0 auto;
+}
+
+.program-container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.programs {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  margin-top: 15px;
+  background-color: #393c43b3;
+  padding: 10px;
+  gap: 10px;
+}
+
+.programs:hover {
+  border: 1px solid var(--light-color);
+  cursor: pointer;
+  transform: translateY(1rem);
+  transition: transform 200ms ease-in-out;
+}
+
+.programs i {
+  font-size: 3rem;
+  color: var(--light-color);
+}
+
+.program-title {
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 22px;
+  color: var(--primary-color);
+  text-transform: capitalize;
+}
+
+.program-dets {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 22px;
+  color: var(--light-color);
+  text-transform: capitalize;
+}
+
+.btn {
+  background-color: var(--primary-color);
+  color: var(--light-color);
+  text-transform: capitalize;
+  margin-top: 1em;
+  padding: 1.5rem 3rem;
+  cursor: pointer;
+}
+
+.btn:hover,
+.btn:focus {
+  background-color: var(--light-color);
+  color: var(--primary-color);
+}
+
+#performing-artist {
+  padding: 1em 1em;
+}
+
+#performing-artist h2 {
+  display: flex;
+  justify-content: center;
+  color: var(--secondary-color);
+}
+
+.artist-container {
+  margin-top: 1rem;
+  display: grid;
+  gap: 20px;
+}
+
+.artists {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 20px;
+}
+
+.artist-img img:nth-child(1) {
+  width: 80px;
+  height: 80px;
+}
+
+.artist-img img:nth-child(2) {
+  width: 120px;
+  height: 120px;
+  position: relative;
+  bottom: 69px;
+  left: 9px;
+}
+
+/* .artist-info {
+
+} */
+
+.artist-info .artist-name {
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 22px;
+}
+
+.artist-info .performance-time {
+  font-style: italic;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: var(--primary-color);
+}
+
+.artist-info .artist-desc {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 17px;
+}
+
+#partners {
+  background-color: var(--secondary-color);
+  padding: 1rem 1rem;
+}
+
+.partners-container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 10px;
+  padding: 1rem;
+}
+
+footer {
+  padding: 1em 1em;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+}
```

Comment:
- [ ] Great work implementing the footer :+1: However it would be nice to follow the provided design as requested. Please note the color of the footer on the About Page for the desktop version, it has a dark background, but in your design, it has a white background. Kindly make sure you follow the provided design as much as possible.
### Your footer on the About Page(desktop version)
![Screenshot from 2023-05-23 00-10-56](https://github.com/Othniel2471/capstone-html-css/assets/91142432/6c23742d-9762-48a1-8d2c-1d3ed5c3909c)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-05-23 00-00-15](https://github.com/Othniel2471/capstone-html-css/assets/91142432/59fcf49f-a5d6-47ac-87b8-8bace85ff598)